### DISCUSSION TITLE
Nubee Question

### DISCUSSION THREAD

**Discussion by GeophreeH** [joomengine/Joomla-Component-Builder#777] [Q&A] [answered]
Can I create a component with JCB on a development site, fully test it, then package it up into an pkg zip file and install it on a customer's site?

**Comment by Llewellynvdm** [comment, marked as answer]
Yes, this is how it works, go head and watch the tutorials at https://www.youtube.com/playlist?list=PLQRGFI8XZ_wtGvPQZWBfDzzlERLQgpMRE
